formats/email_alert_signup/publisher/links.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "related"
```

Comment:
Well no, but I thought requiring it would be good so we know which page you're signing up to?